app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -0,0 +1,78 @@
+---
+title: If you have a degree
+related_content:
+    Get support training to teach if you're disabled: "/get-support-training-to-teach-if-you-are-disabled"
+    Improve your subject knowledge : "/improve-your-subject-knowledge"
+    Tips on applying for teacher training : "/tips-on-applying-for-teacher-training"
+---
+
+
+If you have a degree, or you’re currently studying for one, there are different routes you can take to getting [qualified teacher status](/what-is-qts) (QTS). You need QTS to work in the majority of schools in England including state maintained primary, secondary and special schools.
+
+We also have information for people outside of the UK who want to get [international qualified teacher status](/international-qualified-teacher-status) (iQTS) or [come to teach in England](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk).
+
+##Study as a postgraduate student
+You can study for [a PGCE or PGDE](/what-is-a-pgce) (an academic qualification) at a university. This is one way to get QTS. Most of the time you will be in schools on placement but there is some academic learning. Courses typically take one year but you may be able to study part-time.
+
+Fees for full-time courses are usually around £9,250 but you can:
+
+- get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects
+- [apply for a student loan](/funding-your-training#tuition-fee-and-maintenance-loans)
+
+Fees are higher for international students.
+
+You should check the fees for a particular course you are interested in with the course provider.
+
+##Train in school (fee paying)
+There are different programmes that focus on training you while you teach in a school:
+
+- School-centred initial teacher training (SCITT)
+- School Direct (fee-funded)
+
+SCITT programmes are delivered by a network of schools, providing school-based training. You won’t be paid a salary while you are training and will pay fees.
+
+School Direct (fee-funded) programmes are delivered by groups of schools in collaboration with a SCITT or university. The content and structure of a School Direct course is broadly similar to a SCITT course and they don't pay you a salary. Some SCITT and School Direct (fee-funded) programmes give you a PGCE or PGDE in addition to QTS. Courses typically take one year but you may be able to study part-time.
+
+Fees are usually around £9,250 but you can:
+
+- get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects
+- [apply for a student loan](/funding-your-training#tuition-fee-and-maintenance-loans)
+
+You should check the fees for a particular course you are interested in with the course provider.
+
+##Earn a salary while you train
+These courses are usually only suitable for people who already have relevant teaching experience or are already working as unqualified teachers. You may be able to study part-time.
```

Comment:
I'm not sure this applies to Teach First, or potentially even the apprenticeship stuff. I **think** this only applies to the School Direct (salaries). We can clarify this with Phoebe later, but this might need a rejig.

Reply:
I think we should perhaps also say something about salaried courses being limited in number (better worded than that) - UR consistently shows that people have the wrong expectations about this